common/src/main/java/com/microsoft/identity/common/internal/broker/BrokerRequest.java

```diff
@@ -118,6 +127,13 @@
     @SerializedName(SerializedNames.EXTRA_QUERY_STRING_PARAMETER)
     private String mExtraQueryStringParameter;
 
+    /**
+     * Extra options flags for the request.
+     */
+    @Nullable
+    @SerializedName(SerializedNames.EXTRA_OPTIONS)
+    private String mExtraOptions;
```

Comment:
What is this 'extraOptions' for?

Reply:
Right now I'm just using it to induce the v3 flow on request.  Properties bags are useful for all sorts of things, though.

Reply:
And, once again, I can ditch it if it's objectionable, or will cause problems.